it/v1/endpoints/RetrievePeriodSummaryControllerISpec.scala

```diff
@@ -280,20 +280,6 @@ class RetrievePeriodSummaryControllerISpec extends IntegrationBaseSpec {
     }
 
     "return error according to spec" when {
-      "tys validation" when {
-        "validation fails with RuleTaxYearNotSupported error" in new TysTest {
-
-          override def setupStubs(): StubMapping = {
-            AuditStub.audit()
-            AuthStub.authorised()
-            MtdIdLookupStub.ninoFound(nino)
-          }
-
-          val response: WSResponse = await(invalidTaxYearRequest("2021-22").get())
```

Comment:
The `invalidTaxYearRequest` method can now be removed as it isn't used.

Reply:
Fixed